training/cloud/Containerfile

```diff
@@ -0,0 +1,8 @@
+ARG BOOTC_IMAGE=registry.stage.redhat.io/rhelai1/bootc-nvidia-rhel9:latest
```

Comment:
The other Containerfiles have a generic image name here. Putting Red Hat is kind of limiting.
```suggestion
ARG BOOTC_IMAGE=quay.io/ai-labs/bootc-nvidia:latest
```

Reply:
I wanted to use the official RHEL AI image as a sane default, to make it easier for customers that need to come here and run it

At least for now, until we're delivering official images for all the clouds